src/js/routes/index.js

```diff
@@ -14,6 +16,7 @@ import services from "../../../plugins/services/src/js/routes/services";
 import settings from "./settings";
 import systemOverview from "./system-overview";
 import catalog from "./catalog";
+import development from "./development";
```

Comment:
We could use require instead within the condition?

Reply:
I think we fixed this in newer commits, let me check

Reply:
I checked and with the current solution we don't include anything GraphiQL realted into the bundle, so the problem is fixed

Reply:
you're right in that we don't have a big problem here having lazy-loaded graphiQL. 

i still think it would preferable to not ship anything development-related at all in production. can we come up with a clean solution for that here, so we want to reuse it in the future?